common/ManagementTestShared/Redesign/AddDeleteAfterTagPolicy.cs

```diff
@@ -10,6 +10,8 @@
 using System.Text.Json;
 using System.Text.RegularExpressions;
 
+#nullable disable
```

Comment:
This was necessary because Azure.Provisioning has nullable references enabled currently.